src/pages/docs/channels/options/deltas.mdx

```diff
@@ -1,99 +1,93 @@
 
-  h3(#install). Installation from npm for Node.js
+  ### Installation from npm for Node.js <a id="install"/>
 
-  ```[bash]
+  <Code fixed="true">
+  ```shell
   npm install @ably/vcdiff-decoder
   ```
+  </Code>
 
   and require as:
 
-  ```[javascript]
+  <Code>
+  ```javascript
   const vcdiffPlugin = require('@ably/vcdiff-decoder');
   ```
 
-  ```[nodejs]
+  ```nodejs
   const vcdiffPlugin = require('@ably/vcdiff-decoder');
   ```
+  </Code>
 
-  h3(#browsers). Script include for web browsers
+  ### Script include for web browsers <a id="browsers"/>
 
   Include the library in your HTML from our CDN:
 
-  ```[html]
+  <Code fixed="true">
+  ```javascript
   <script src="https://cdn.ably.io/lib/vcdiff-decoder.min-1.js"></script>
   ```
+  </Code>
 
-  h3(#exported). Exported functions
+  ### Exported functions <a id="exported"/>
 
   The vcdiff decoder library exports the following function for manual delta decoding.
 
-  @decode(delta, source)@ applies a vcdiff delta to a source message to return a "@Uint8Array@":https://nodejs.org/api/buffer.html#buffer containing the target message:
-
-  * @delta@: The binary delta/diff data.
-  * @source@: The original message to apply the delta to.
+  `decode(delta, source)` applies a vcdiff delta to a source message to return a [`Uint8Array`](https://nodejs.org/api/buffer.html#buffer) containing the target message:
 
-blang[default].
+  * `delta`: The binary delta/diff data.
+  * `source`: The original message to apply the delta to.
+</If>
 
-h2(#subscribe). Subscribe using delta
+## Subscribe using delta <a id="subscribe"/>
 
-Set the @delta@ property of @params@ to @vcdiff@ in order to enable deltas when subscribing to a channel.
+Set the `delta` property of `params` to `vcdiff` in order to enable deltas when subscribing to a channel.
 
-This will cause delta messages to be generated by the server and sent to the client, and the library reconstitutes the original message payload. Messages on the channel are delivered to the subscriber's listener in the same way as with a normal "subscription":/docs/pub-sub#subscribe.
+This will cause delta messages to be generated by the server and sent to the client, and the library reconstitutes the original message payload. Messages on the channel are delivered to the subscriber's listener in the same way as with a normal [subscription](/docs/pub-sub#subscribe).
 
-Note that in some SDKs, the @vcdiff@ delta decoding library is excluded from the default distribution in order to avoid increasing the size. In these cases, it is also necessary to supply the delta decoder plugin when instantiating Ably.
+Note that in some SDKs, the `vcdiff` delta decoding library is excluded from the default distribution in order to avoid increasing the size. In these cases, it is also necessary to supply the delta decoder plugin when instantiating Ably.
 
-```[realtime_javascript]
+<Code>
```

Comment:
This block has an issue rendering on first load... I wonder if it's because of `<If>` and how that then processes `realtime_`

Reply:
But this code isn't in an `<If>`

Reply:
It isn't, but I wonder if the `<If>` doesn't account for the language prefix, as we haven't had to deal with them and Ifs before

Reply:
`If` doesn't take the SDK prefix into account

Reply:
Jamie's fix worked to resolve this.